zun_kata.md

```diff
@@ -0,0 +1,117 @@
+# Introduction
+
+This guide describes how to get Kata containers to work with OpenStack Zun using DevStack on Ubuntu 16.04. Running DevStack with this guide will setup Docker and Clear Containers 2.0, which you replace with Kata Containers. Currently, the instructions are based on the following links:  
```

Comment:
Could you split this into separate lines. The carriage return won't be applied when the md file is rendered, but it's easier to review.

Reply:
s/Kata containers/Kata Containers/